src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticSourceAdapter.cs

```diff
@@ -79,7 +125,7 @@ public void Write(string diagnosticName, object parameters)
                 return;
             }
 
-            if (_listener.IsEnabled != null && !_listener.IsEnabled(diagnosticName))
+            if (_listener.IsEnabled != null && !_listener.IsEnabled(diagnosticName, null, null))
```

Comment:
Do you think [this](https://github.com/aspnet/EventNotification/blob/dev/src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticSourceAdapter.cs#L149) can now be a part of the event context instead of a class?